include/nbl/builtin/hlsl/spirv_intrinsics/core.hlsl

```diff
@@ -110,10 +110,6 @@ void memoryBarrier(uint32_t memoryScope, uint32_t memorySemantics);
 #pragma region CONVERSION
 
 template<class T, class U>
-// [[vk::ext_extension("SPV_KHR_physical_storage_buffer")]]
-// [[vk::ext_capability(spv::CapabilityPhysicalStorageBufferAddresses)]]
-// [[vk::ext_capability(spv::CapabilityAddresses)]]
```

Comment:
you'd probably be declaring those on a partial specialization at some point, useful to leave that as a comment